Dribbblish-dynamic/README.md

```diff
@@ -0,0 +1,84 @@
+# Dribbblish-dynamic
+
+## Screenshots
+#### Dark
+![demo1](./color-match-bg.gif)
```

Comment:
Add preview for other color schemes as well

Reply:
Actually I never tried other schemes. I will test them and add previews

Reply:
done